wscript

```diff
@@ -659,14 +659,9 @@ video_output_features = [
         'deps': 'vaapi && gl-wayland',
         'func': check_pkg_config('libva-wayland', '>= 1.1.0'),
     }, {
-        'deps': 'vaapi-wayland && memfd_create',
-        'name': 'vaapi-wayland-memfd',
-        'desc': 'Wayland vaapi support',
-        'func': check_true,
-    }, {
-        'deps': 'wayland && memfd_create',
         'name': 'dmabuf-wayland',
         'desc': 'Wayland dmabuf support',
+        'deps': 'wayland && memfd_create && (vaapi || drm)',
```

Comment:
Same as the previous comments about this belong to the previous commit. Also could you make the dep `vaapi-wayland` instead of `vaapi` (to match meson)?